src/Login.js

```diff
@@ -4,27 +4,14 @@ import Stytch from '@stytch/stytch-js'
 const Login = () => {
 	React.useEffect(() => {
 		// Initialize Stytch.js with your public token. You can find this in your Stytch dashboard under API Keys.
-		var STYTCH_PUBLIC_TOKEN = "public-token-test-111111111";
+		var STYTCH_PUBLIC_TOKEN = "public-token-test-196bf55b-4d4d-4694-bb9c-4312a2398cf6";
```

Comment:
Can we keep this public token generic?

Reply:
whoops, https://github.com/stytchauth/stytchjs-react-express/pull/2/commits/3668117a06d5d5b8a9d0aed779f5979e802c278d